### DISCUSSION TITLE
Latest Swiper updates seems to be breaking the swipe function

### DISCUSSION THREAD

**Discussion by processedfood** [nolimits4web/swiper#7635] [Q&A]
I am using the following script in my Webflow page header. It seems that the swipe function is broken. Anyone experienced this?

<script src="https://unpkg.com/swiper/swiper-bundle.min.js"></script>
<link rel="stylesheet" href="https://unpkg.com/swiper/swiper-bundle.min.css">

https://github.com/user-attachments/assets/b26a3609-e4e3-457e-81e3-257300aa2040

**Comment by gekkyo** [comment]
After version 11.1.6 has released, I'm experiencing the same issue.

**Comment by OSMANCAKlR** [comment]
Yep, experiencing the same issue. Anyone know a fix?

**Comment by michaelbourne** [comment]
For curiosities sake, can you try it with `touchEventsTarget: 'container'`? I had a similar issue and this fixed it, but I cant rule out if it was unrelated.
